lib/tableau_api/resources/workbooks.rb

```diff
@@ -55,8 +55,29 @@ def publish(name:, project_id:, file:, overwrite: false, show_tabs: false, conne
 
       CAPABILITY_MODES = %w(ALLOW DENY).freeze
 
+      def permissions(workbook_id:)
+        res = @client.connection.api_get("sites/#{@client.auth.site_id}/workbooks/#{workbook_id}/permissions")
+
+        raise TableauError, res if res.code != 200
+        permissions = HTTParty::Parser.new(res.body, :xml).parse['tsResponse']['permissions']['granteeCapabilities']
+        permissions.map do |p|
+          grantee_type = p['group'].nil? ? 'user' : 'group'
+
+          capabilities = {}
+          p['capabilities']['capability'].each do |c|
+            capabilities[c['name']] = c['mode'] == 'Allow'
+          end
+
+          {
+            'granteeType' => grantee_type,
+            'granteeId' => p[grantee_type]['id'],
+            'capabilities' => capabilities
```

Comment:
Do you want to make these keys symbols and snake_case for consistency with `add_permissions`?